### ISSUE TITLE
A11y_30days_AI4W:Ensure links are distinguished from surrounding text in a way that does not rely on color.

### ISSUE THREAD

**Issue by tripathiakhil429** [microsoft/30daysof#189]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**GitHub Tags:**
#A11yTCS;#A11yMAS;#A11ySev2;#30days;#P3_30days_Dec22;#WCAG1.4.1;#DesktopWeb;#Win11;#ChromiumEdge;#WCP;#AI4W

### Environment Details:
Application Name: 30days
#URL: https://microsoft.github.io/30daysof/blog/tags/30-days-of-data-science/page/7/
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)

### Repro Steps:
Hit the URL: https://microsoft.github.io/30daysof/blog/tags/30-days-of-data-science/page/7/
Run fast pass.
### Issue:
Ensure links are distinguished from surrounding text in a way that does not rely on color (link-in-text-block - https://dequeuniversity.com/rules/axe/4.6/link-in-text-block?application=msftAI)

Target application: 8 posts tagged with "30DaysOfDataScience" | #30DaysOf Learning - https://microsoft.github.io/30daysof/blog/tags/30-days-of-data-science/page/7/

Element path: a[href$="rss.xml"]

Snippet: <a href="/30daysof/blog/rss.xml">feed</a>

How to fix: 
Fix any of the following:
  The link has insufficient color contrast of 1.64:1 with the surrounding text. (Minimum contrast is 3:1, link text: #4f2e85, surrounding text: #1c1e21)
  The link has no styling (such as underline) to distinguish it from the surrounding text

Environment: Microsoft Edge version 110.0.1587.50

====

This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
![image](https://user-images.githubusercontent.com/85679252/220861821-ecd1c968-15eb-40d4-a8cf-a08f81e8d880.png)

**Comment by tripathiakhil429** [comment]
This bug is logged as part of WCP scan https://webcompliance.trafficmanager.net/ScanResultsApi/GetAccessibilityScanConsolidatedReport?siteId=9a9e81df-26b8-4f2f-1ae9-08da8f35e51d

**Comment by nitya** [comment]
@tripathiakhil429 we have decided to unpublish the site and deactivate it from internal listings. Can you verify and close this issue now that website is no longer active?

**Comment by tripathiakhil429** [comment]
As the Site is now not to be published , closing this bug.

**Comment by tripathiakhil429** [comment]
**Github tags**:#Closed
